### ISSUE TITLE
How to use custom fonts (fontFamily) with Hyper.js (Windows 10 + cmd.exe shell)? 

### ISSUE THREAD

**Issue by mrjayviper** [zeit/hyper#3793]
Issue: I cannot use custom fonts on Hyper.js + Windows 10.

I have commented out the default fontFamily line and replaced it with mine and all I get some ugly display. I don't get the problem with the default fontFamily. This is the case for all the custom fontFamily I've tried (see my config below)

Any ideas on how to fix?

Here's a [screenshot](https://i.imgur.com/rSWDUpU.png)

edit: I just installed ConEmu and changed the Font to use "Fira Code Light" and "InputMono" (which I also tried on Hyper) and both worked fine.

----------------------

version: 3.0.2. stable
.hyper.js: https://gist.github.com/mrjayviper/3742bed879f99f1ba0078be496be4fb5
shell: cmd.exe (though I don't think this should affect my issue?)

I installed (for all users) 3 font families: InputMono, Fira Code and nerdfont mononoki. As a test, I also tried using the built-in font called "Cambria Bold"

Looking at Windows Settings => Personalization => Fonts => Selected Font Properties => Metadata (These are just samples. There are more fonts for the 4 families)

Cambria Bold: Fullname = Cambria Bold
Fira Code Regular: Fullname = Fira Code Regular
Fira Code Light: Fullname = Fira Code Light
Input Mono Regular : Fullname = InputMono
Input Mono Light Italic : Fullname = InputMono LightIta
mononoki Nerd Font: Fullname = mononoki-Regular Nerd Font Complete

**Comment by GeorgeWalt** [comment]
In Hyper, click on the top left icon, goto Edit -> Preferences, and there you can change your active font and fallback Fonts.
It looks something like this:

`// font family with optional fallbacks
 fontFamily: 'Menlo, "DejaVu Sans Mono", Consolas, "Lucida Console", monospace',`

**Comment by Akhila-C-R** [comment]
I wanted to add the Ubuntu Mono font and it's not working on hyper

**Comment by utsav0904** [comment]
It's working fine for me, just add Ubuntu Mono as a prefix like below

 fontFamily: 'Ubuntu Mono,Menlo, "DejaVu Sans Mono", Consolas, "Lucida Console",monospace',

**Comment by utsav0904** [comment]
It's working fine for me, just add Ubuntu Mono as a prefix like below

fontFamily: 'Ubuntu Mono,Menlo, "DejaVu Sans Mono", Consolas, "Lucida Console",monospace',

**Comment by sbanders1** [comment]
@mrjayviper I was wrestling with this today on Hyepr.js with Windows 10 Pro and I figured out my issue. 

For others who might be wrestling with this, I also was having issues where the font would create tiny boxes with question marks. Turns out it was because my version of Windows 10 Pro was not activated. I found this out by trying to search for Fonts in the settings, and was unable to search for any of the Nerd Fonts I thought I had downloaded. Activating Windows 10 Pro fixed the problem as soon as I reloaded. My use case was trying to get `oh-my-posh` to work.

I made sure:
* Windows 10 Pro was activated
* fonts I wanted were downloaded and appeared when I called `fc-list | grep MesloLGS` for example
* `fontFamily` in hyper settings was: fontFamily: '"MesloLGS NF", "Fira Code", "FiraCode NF", monospace',
* locale was `UTF-8` or `en_US.UTF-8`
* After running `echo $(brew --prefix oh-my-posh)` to find where the themes were located, running `eval $(oh-my-posh init shell bash --config agnoster.omp.json)` on the theme I wanted
